README.md

```diff
@@ -124,6 +124,11 @@ If you get the error `django.db.utils.OperationalError: FATAL:  role "postgres"
 
 ### Releasing to production
 
+>#### :warning: Release to production outside of business hours
+> __Business hours__: 09:00 to 20:00  
+>__Why?__ Any downtime on the frontend and backend between 07:00 and 20:00 - can have serious consequences, leading to shut down of the court legal advice centres, possible press reports and maybe MP questions  
```

Comment:
This line mentions 7 am as the period start and the above mentions 9 am. Which one is correct?